From 5b9d53d984733c60ac8071abe9d261b82db62937 Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Wed, 14 Aug 2019 12:10:56 +0100
Subject: [PATCH 1/2] appdata: simplify OARS data

If the value is 'none', the category can be omitted.
---
 data/org.gnome.Characters.appdata.xml.in | 30 +-----------------------
 1 file changed, 1 insertion(+), 29 deletions(-)

diff --git a/data/org.gnome.Characters.appdata.xml.in b/data/org.gnome.Characters.appdata.xml.in
index 1ff0e6a..19486e0 100644
--- a/data/org.gnome.Characters.appdata.xml.in
+++ b/data/org.gnome.Characters.appdata.xml.in
@@ -32,33 +32,5 @@
     <kudo>SearchProvider</kudo>
   </kudos>
   <translation type="gettext">org.gnome.Characters</translation>
-  <content_rating type="oars-1.1">
-    <content_attribute id="violence-cartoon">none</content_attribute>
-    <content_attribute id="violence-fantasy">none</content_attribute>
-    <content_attribute id="violence-realistic">none</content_attribute>
-    <content_attribute id="violence-bloodshed">none</content_attribute>
-    <content_attribute id="violence-sexual">none</content_attribute>
-    <content_attribute id="violence-desecration">none</content_attribute>
-    <content_attribute id="violence-slavery">none</content_attribute>
-    <content_attribute id="violence-worship">none</content_attribute>
-    <content_attribute id="drugs-alcohol">none</content_attribute>
-    <content_attribute id="drugs-narcotics">none</content_attribute>
-    <content_attribute id="drugs-tobacco">none</content_attribute>
-    <content_attribute id="sex-nudity">none</content_attribute>
-    <content_attribute id="sex-themes">none</content_attribute>
-    <content_attribute id="sex-homosexuality">none</content_attribute>
-    <content_attribute id="sex-prostitution">none</content_attribute>
-    <content_attribute id="sex-adultery">none</content_attribute>
-    <content_attribute id="sex-appearance">none</content_attribute>
-    <content_attribute id="language-profanity">none</content_attribute>
-    <content_attribute id="language-humor">none</content_attribute>
-    <content_attribute id="language-discrimination">none</content_attribute>
-    <content_attribute id="social-chat">none</content_attribute>
-    <content_attribute id="social-info">none</content_attribute>
-    <content_attribute id="social-audio">none</content_attribute>
-    <content_attribute id="social-location">none</content_attribute>
-    <content_attribute id="social-contacts">none</content_attribute>
-    <content_attribute id="money-purchasing">none</content_attribute>
-    <content_attribute id="money-gambling">none</content_attribute>
-  </content_rating>
+  <content_rating type="oars-1.1"/>
 </component>
-- 
2.20.1

